bundles/org.openhab.core.compat1x/src/main/java/org/openhab/core/binding/internal/BindingConfigReaderDelegate.java

```diff
@@ -38,9 +38,10 @@ public void validateItemType(String itemType, String bindingConfig) throws Bindi
         try {
             reader.validateItemType(getOpenHABItem(itemType), bindingConfig);
         } catch (org.openhab.model.item.binding.BindingConfigParseException e) {
-            throw new BindingConfigParseException(e.getMessage());
```

Comment:
Why not just something like `throw new BindingConfigParseException(e.getMessage(), e);` it looks more standard to me.

Reply:
Agreed.